bookshelf.js

```diff
@@ -149,10 +150,22 @@ Bookshelf.initialize = function(knex) {
   });
   Model.fetchAll = function(options) { return this.forge().fetchAll(options); };
 
-  Model.extend = Collection.extend = require('simple-extend');
+  Model.extend = function(protoProps, staticProps){
+    // Set per-model Error
+    ['NotFoundError', 'NoRowsUpdatedError', 'NoRowsDeletedError'].forEach(function(errorType){
+      this[errorType] = createError(BookshelfModel[errorType],
+      _.capitalize((protoProps.tableName) || '') + errorType);
+    }.bind(this));
+    return require('simple-extend').apply(this, [protoProps, staticProps])
+  };
+  Collection.extend = function(protoProps, staticProps){
+    this.EmptyError = createError(BookshelfCollection.EmptyError,
+      _.capitalize(protoProps.tableName) + "EmptyError");
```

Comment:
Looks like the bookshelf is using single quotes so you probably want to replace double quotes here